src/main/java/com/jcabi/github/Issues.java

```diff
@@ -85,4 +87,116 @@ Issue create(
     Iterable<Issue> iterate(@NotNull(message = "map of params can't be NULL")
         Map<String, String> params);
 
+    /**
+     * Search for issues within the given repository.
+     *
+     * @param sort The sort field
+     * @param direction The sort direction
+     * @param qualifiers The search qualifier
+     * @return Issues
+     * @throws java.io.IOException If there is any I/O problem
+     * @since 0.22.0
+     * @see <a href="https://developer.github.com/v3/issues/#list-issues-for-a-repository">List issues for a repository</a>
+     */
+    @NotNull(message = "Iterable of issues is never NULL")
+    Iterable<Issue> search(
```

Comment:
let's return `java.util.List` instead of `Iterable`, I guess when one searches for something he or she would prefer to have a collection object.